source/extensions/extensions_metadata.yaml

```diff
@@ -0,0 +1,579 @@
+  status: wip
+envoy.tracers.datadog:
+  categories:
+  - envoy.tracers
+  security_posture: robust_to_untrusted_downstream
+envoy.tracers.dynamic_ot:
+  categories:
+  - envoy.tracers
+  security_posture: robust_to_untrusted_downstream
+envoy.tracers.lightstep:
+  categories:
+  - envoy.tracers
+  security_posture: robust_to_untrusted_downstream
+envoy.tracers.opencensus:
+  categories:
+  - envoy.tracers
+  security_posture: robust_to_untrusted_downstream
+envoy.tracers.skywalking:
+  categories:
+  - envoy.tracers
+  security_posture: robust_to_untrusted_downstream
+  status: wip
+envoy.tracers.xray:
+  categories:
+  - envoy.tracers
+  security_posture: robust_to_untrusted_downstream
+envoy.tracers.zipkin:
+  categories:
+  - envoy.tracers
+  security_posture: robust_to_untrusted_downstream
+envoy.transport_sockets.alts:
+  categories:
+  - envoy.transport_sockets.downstream
+  - envoy.transport_sockets.upstream
+  security_posture: robust_to_untrusted_downstream_and_upstream
+envoy.transport_sockets.raw_buffer:
+  categories:
+  - envoy.transport_sockets.downstream
+  - envoy.transport_sockets.upstream
+  security_posture: requires_trusted_downstream_and_upstream
+envoy.transport_sockets.starttls:
+  categories:
+  - envoy.transport_sockets.downstream
+  - envoy.transport_sockets.upstream
+  security_posture: robust_to_untrusted_downstream_and_upstream
+envoy.transport_sockets.tap:
+  categories:
+  - envoy.transport_sockets.downstream
+  - envoy.transport_sockets.upstream
+  security_posture: requires_trusted_downstream_and_upstream
+  status: alpha
+envoy.transport_sockets.tls:
+  categories:
+  - envoy.transport_sockets.downstream
+  - envoy.transport_sockets.upstream
+  security_posture: robust_to_untrusted_downstream_and_upstream
+envoy.transport_sockets.upstream_proxy_protocol:
+  categories:
+  - envoy.transport_sockets.upstream
+  security_posture: robust_to_untrusted_downstream_and_upstream
+envoy.upstreams.http.generic:
+  categories:
+  - envoy.upstreams
+  security_posture: robust_to_untrusted_downstream
+envoy.upstreams.http.http:
+  categories:
+  - envoy.upstreams
+  security_posture: robust_to_untrusted_downstream
+envoy.upstreams.http.http_protocol_options:
+  categories:
+  - envoy.upstreams
+  security_posture: robust_to_untrusted_downstream
+envoy.upstreams.http.tcp:
+  categories:
+  - envoy.upstreams
+  security_posture: robust_to_untrusted_downstream
+envoy.upstreams.tcp.generic:
+  categories:
+  - envoy.upstreams
+  security_posture: robust_to_untrusted_downstream
+envoy.wasm.runtime.null:
+  categories:
+  - envoy.wasm.runtime
+  security_posture: unknown
+  status: alpha
+envoy.wasm.runtime.v8:
+  categories:
+  - envoy.wasm.runtime
+  security_posture: unknown
+  status: alpha
+envoy.wasm.runtime.wasmtime:
+  categories:
+  - envoy.wasm.runtime
+  security_posture: unknown
+  status: alpha
+envoy.wasm.runtime.wavm:
+  categories:
+  - envoy.wasm.runtime
+  security_posture: unknown
+  status: alpha
```

Comment:
There were a bunch of comments around these status, i.e. that this alpha/unknown status can't change until we do deliberate review and change to threat model, for Wasm runtimes. Can you move these as well to this file? Also pleaes check we haven't missed other places there are similar comments we are missing.

Reply:
this PR hasnt removed the other comments yet - but yep - i will go through and add any comments i can find from the existing metadata registration

Reply:
@htuch i have gone through all of the `BUILD` files with wasm in the path (and quite a few of the others) and none contain comments

can you point me to any specific examples please ?

Reply:
Hah, not merged yet (I thought it had), see https://github.com/envoyproxy/envoy/pull/16057/files#diff-85815a26c8f765868f9c242a1968320d41e9c5798cbe94358339f3d99d0a70c2R16. We will need to update that PR.